Template.php

```diff
@@ -1092,7 +1092,7 @@ public function get_identifiers_from_url($url_sent = NULL) {
         }
         if ($this->wikiname() === 'cite web') $this->change_name_to('cite arxiv');
         
-      } elseif (preg_match("~https?://(?:www\.|)ncbi\.nlm\.nih\.gov/(?:m/)?(?:pubmed/|entrez/eutils/elink\.fcgi\S+dbfrom=pubmed\S+/|entrez/query\.fcgi\S+db=pubmed\S+).*?=?(\d+)~i", $url, $match)) {
+      } elseif (preg_match("~https?://(?:www\.|)ncbi\.nlm\.nih\.gov/(?:m/)?(?:pubmed/|entrez/eutils/elink\.fcgi\S+dbfrom=pubmed\S+/|entrez/query\.fcgi\S+db=pubmed\S+|pmc/articles/pmid/).*?=?(\d+)~i", $url, $match)) {
```

Comment:
Could we improve readability by having one line per option?
```suggestion
      } elseif (preg_match("~https?://(?:www\.|)ncbi\.nlm\.nih\.gov/(?:m/)?" 
      . "(?:pubmed/|" 
      . "/eutils/elink\.fcgi\S+dbfrom=pubmed\S+/|"
      . "entrez/query\.fcgi\S+db=pubmed\S+|"
      . "pmc/articles/pmid/)"
      . ".*?=?(\d+)~i", $url, $match)) {
```